magento/magento2
# Fix #10417

When adding images in CMS Pages / Blocks or Catalog and Category the path to the image is incorrect. When you insert image in 2.1.8, we get ```<p><img src="{{media url="wysiwyg/Hond.jpg"}}" width="1024" height="683" /></p>``` when we insert image in 2.1.9 ```<p><img src="http://local.magento/admin/cms/wysiwyg/directive/___directive/e3ttZWRpYSB1cmw9Ind5c2l3eWcvVmVsZC5qcGcifX0%2C/key/d3f89696a93152f6d352e5d5cb391180a27c8a7924d9fe3b583d3e9f03d00753/" width="959" height="1024" /></p>```

The issue is also a problem in 2.2-develop. This fix is for 2.2-develop. Will create backport voor 2.1-develop branch.

### Fixed Issues (if relevant)
1. https://github.com/magento/magento2/issues/10417

### Manual testing scenarios
1. Go to CMS Pages.
2. Add image
3. Check the path (show/hide editor)

### Contribution checklist
